doctypes/tile.md

```diff
@@ -0,0 +1,135 @@
+# Tile doctype
+
+> Tile documents used to describe schemas, metadata, services, access control, etc.
+
+Tiles are a flexible way to create single or multi-party statements and can be used to define infrastructure descriptions, service agreements, routing information, data schema definitions, access control permissions, and much more.
+
+## Index
+
+- [Genesis record](#genesis-record)
+- [Signed record](#signed-record)
+- [Update rules](#update-rules)
+- [Examples](#examples)
+    - [Collection tile](#collection-tile)
+    - [Service tile](#service-tile)
+    - [Privacy tile](#privacy-tile)
+
+## Genesis record
+
+**Doctype name:** `tile`
+
+The genesis record of a tile document is quite flexible. The `content` property contains the data of the tile and can be set to any json data. The `owners` property must contain an array of DIDs (at least one). It specifies the `owners` who are allowed to update the document (note that a signature from all the owners is needed in order to make a valid update to a tile). In the example below the 3ID DID method is used, but Ceramic can support other DID methods as well.
+
+Tile genesis records are stored in IPFS using the [`dag-cbor`](https://github.com/ipld/js-ipld-dag-cbor/) IPLD format.
+
+```JSON
+{
+  "doctype": "tile",
+  "owners": ["did:3:<CID>"],
+  "content": ...
+}
+```
+
+## Signed record
+
+Signed records in 3ID are stored in IPFS using the `dag-jose` or `dag-cose` formats. These formats allow for IPLD objects to be stored along with a standardized signature format. They also allow DAG objects to be encrypted. The update to the document is encoded using [json-patch](https://github.com/Starcounter-Jack/JSON-Patch).
+
+```JSON
+{
+  "next": <CID-previous-record>,
+  "content": <JSON-patch-object>
+}
+```
+
+## Update rules
+
+An update to a tile can change anything in the `content` of the tile document. If there are multiple owners only the first *signed record* needs to contain the json-patch that changes the document. The other owners can simply sign a record that only contains the `next` pointer to the previous signed record.
+
+## Examples
+
+Below there are three examples of possible tiles, Collection, Service, and Privacy tiles. These tiles in particular are useful when specifying a user cenrtic data storage system.
```

Comment:
Change "Collection, Service, and Privacy tiles" to: "Collection Policies, Service Policies, and Privacy Policies".